docs/sphinx/source/whatsnew/v0.6.1.rst

```diff
@@ -24,6 +24,11 @@ API Changes
 * Changed function name from :py:func:`pvlib.solarposition.get_rise_set_transit` (deprecated)
   to :py:func:`pvlib.solarposition.sun_rise_set_transit_spa. `sun_rise_set_transit_spa`
   requires time input to be localized to the specified latitude/longitude. (:issue:`316')
+* Add `max_airmass` keyword argument to :py:func:`pvlib.irradiance.disc`.
+  Default value (`max_airmass=12`) is consistent with polynomial fit in
+  original paper describing the model. Output of
+  :py:func:`pvlib.irradiance.dirint` and `:py:func:`pvlib.irradiance.dirindex`,
+  are different when `min_cos_zenith` and `max_zenith` kwargs are used.
```

Comment:
This second sentence is not clear to me.  Are you saying that the two functions produce different output with the same kwarg values, or that each function's output is different if the kwarg is set different from its default?

Reply:
The outputs of each of these functions will be different than they were before if the user set the kwargs to more permissive values.

Reply:
to be clear, this is only because of the change in `disc` and not because I've directly changed anything in these other functions.

Reply:
Perhaps "Changing the value(s) of `min_cos_zenith` or `max_zenith` kwargs will change the output of  :py:func:`pvlib.irradiance.dirint` and `:py:func:`pvlib.irradiance.dirindex`.